### DISCUSSION TITLE
Limitation on Maximum Number of Digits

### DISCUSSION THREAD

**Discussion by aditgege** [dm4t2/vue-currency-input#420] [Q&A]
## Issue with Maximum Digits for Integer Input

Hi there,

I'm currently using the Vue Currency Input library in my project, and I’ve encountered an issue related to the maximum number of digits allowed for input values.

#### Configuration:
```javascript
{
  currency: "IDR",
  locale: "en",
  currencyDisplay: "hidden",
  hideGroupingSeparatorOnFocus: true,
  hideCurrencySymbolOnFocus: true,
  hideNegligibleDecimalDigitsOnFocus: true,
  autoDecimalDigits: false,
  autoSign: true,
  useGrouping: true,
  accountingSign: false,
  precision: { min: 0, max: 5 },
}
```

With the above configuration, I’m able to input values like 10000000000.11111 without any issues. However, when attempting to input 1000000000000000, the input field does not accept it. It only accepts up to 10000000000.

Even though I have set the precision to allow 5 decimal digits, I am not using decimal digits in my input. I would expect to be able to input up to 16 integer digits, but the input is limited to 11 integer digits.

Is there something I need to change in my configuration to allow input of 16 integer digits? I’m a bit confused about how the significant digits are being managed and would appreciate any guidance on this matter.

**Comment by dm4t2** [comment]
see #346
